### PR TITLE
Rename mingw-w64-git-credential-manager-core

### PR DESCRIPTION
**Pull request by dscho**
This PR drops the `-core` suffix from the package name, to reflect the change of the original project's name.

This PR needs to wait for https://github.com/git-for-windows/build-extra/pull/396 to be merged, as it is based on that PR's branch.

### CODE REVIEW

File: installer/install.iss [hunk 1]
```diff
@@ -2280,24 +2280,24 @@ begin
 
     GitCredentialManagerPage:=CreatePage(PrevPageID,'Choose a credential helper','Which credential helper should be configured?',TabOrder,Top,Left);
 
-    // Git Credential Manager Core
-    RdbGitCredentialManager[GCM_Core]:=CreateRadioButton(GitCredentialManagerPage,'Git Credential Manager Core','<RED>(NEW!)</RED> Use the new, <A HREF=https://github.com/microsoft/Git-Credential-Manager-Core>cross-platform version of the Git Credential Manager</A>.'+#13+'See more information about the future of Git Credential Manager <A HREF=https://github.com/microsoft/Git-Credential-Manager-Core/blob/master/docs/faq.md#about-the-project>here</A>.',TabOrder,Top,Left);
+    // Git Credential Manager
+    RdbGitCredentialManager[GCM]:=CreateRadioButton(GitCredentialManagerPage,'Git Credential Manager','Use the <A HREF=https://github.com/GitCredentialManager/git-credential-manager>cross-platform Git Credential Manager</A>.'+#13+'See more information about the future of Git Credential Manager <A HREF=https://github.com/GitCredentialManager/git-credential-manager/blob/HEAD/docs/faq.md#about-the-project>here</A>.',TabOrder,Top,Left);
```
Comment (mjcheetham) [inline installer/install.iss:2284]: I wonder if it's time to change this blurb? Does it still make sense now that it's no longer next to the older GCM4W?

File: installer/install.iss [hunk 2]
```diff
@@ -846,12 +846,12 @@ begin
...
                         case Value of
-                            'manager': RecordInferredDefault('Use Credential Manager','Core');
-                            'manager-core': RecordInferredDefault('Use Credential Manager','Core');
+                            'manager': RecordInferredDefault('Use Credential Manager','Enabled');
+                            'manager-core': RecordInferredDefault('Use Credential Manager','Enabled');
                         else
                             begin
-                                if EndsWith(Value, 'manager-core') then
-                                    RecordInferredDefault('Use Credential Manager','Core')
+                                if EndsWith(Value,'manager-core') or EndsWith(Value,'manager') then
```
Comment (mjcheetham) [inline installer/install.iss:853]: So this means both manager and manager-core are resolved to the new GCM? Cool!

**Review by mjcheetham — APPROVED** [review]
Approving (in prep for the other PR to be completed first ofc)

File: installer/install.iss (hunk 2 above)
Comment (dscho) [reply installer/install.iss:853]: Yes!

**Pull request merged**